src/components/Widgets/WordSwipe/WordSwipe.vue

```diff
@@ -72,6 +85,7 @@ export default {
     const evaluateVote = async response => {
       if (props.allSamples[currentSampleId.value].actualValue === response)
         return 1;
+      return 0;
```

Comment:
looks to be declared as an async function but can't see any async calls inside `evaluateVote()`?